molecule/default/converge.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  gather_facts: true
+  vars:
+    - package_source: 'https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.bionic_amd64.deb'
+    - download_dest: "{{ lookup('env','HOME') }}/wkhtmltox_0.12.6-1.bionic_amd64.deb"
+    - package_checksum: sha256:db48fa1a043309c4bfe8c8e0e38dc06c183f821599dd88d4e3cea47c5a5d4cd3
+    - xfonts_package_name: xfonts-75dpi
```

Comment:
Lines 6-10 need to go away. They'll interfere with the role's defaults.

Reply:
Done.